prime-dotnet-webapi/Controllers/SitesController.cs

```diff
@@ -753,6 +753,32 @@ public async Task<ActionResult> SendRemoteUsersEmailUser(int siteId, IEnumerable
             return NoContent();
         }
 
+        // POST: api/Sites/5/site-reviewed-email-user
+        /// <summary>
+        /// Send site reviewed notification email to provider enrolment team
+        /// </summary>
+        /// <param name="siteId"></param>
+        /// <param name="note"></param>
+        /// <returns></returns>
+        [HttpPost("{siteId}/site-reviewed-email-user", Name = nameof(SendSiteReviewedNotificationEmail))]
+        [Authorize(Roles = Roles.ViewSite)]
+        [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+        [ProducesResponseType(StatusCodes.Status403Forbidden)]
+        [ProducesResponseType(typeof(ApiMessageResponse), StatusCodes.Status404NotFound)]
+        [ProducesResponseType(StatusCodes.Status204NoContent)]
+        public async Task<ActionResult> SendSiteReviewedNotificationEmail(int siteId, FromBodyText note)
+        {
+            var site = await _siteService.GetSiteAsync(siteId);
```

Comment:
I would like to stop fetching the entire `Site` just to check that it exists when we can. Looking ahead, the only information on the `Site` we need for this email is the PEC, so we are throwing away a lot of retrieved data here.
I would prefer to see a `SiteService.SiteExists(int siteId)` method, and then only fetching the PEC in `SendSiteReviewedNotificationAsync(int siteId, note)`